frontend/src/app/[locale]/opportunity/[id]/page.tsx

```diff
@@ -41,6 +42,10 @@ export async function generateMetadata({ params }: { params: { id: string } }) {
   return meta;
 }
 
+export function generateStaticParams() {
+  return [];
```

Comment:
> Is that right? Is the idea to try this out and evaluate how it works, or are there plans for next steps, such as as generating a list of all current opportunities at build time and pre-caching? 

Yes. We could certainly to that in the future. We don't have a good way of getting all of the opp ids. We might want an API for that in the future.

> Is the 10 minute window a ballpark estimate at this point that we will analyze in some way after launch and adjust, or is that based on data that leads us to believe it should be a pretty solid permanent setting?

The file links refresh every 30 mins, so that is the higher end. The 10 mins is something we can test and adjust in the future.